contrib_bots/lib/gcalendar.py

```diff
@@ -0,0 +1,333 @@
+from __future__ import print_function
+import httplib2
+import logging
+import os
+import re
+import sys
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from apiclient import discovery, errors
```

Comment:
It seems to be because of [this issue](https://github.com/google/google-api-python-client/issues/299).

I don't know why they provided a workaround and closed the issue, to be honest. Nevertheless, there seems to be [some interest](https://github.com/google/google-api-python-client/issues/325) in fixing it.

The "silencing fix" the project's members suggested doesn't seem to work:

```python
logging.getLogger('googleapicliet.discovery_cache').setLevel(logging.ERROR)
```

But, anyways, it shouldn't be a problem, since we could classify this as "expected behavior".

---

As a side note, one of the comments in the [first issue](https://github.com/google/google-api-python-client/issues/299) I mentioned before says:

> It seems to be the better way to silence this error, if one does not care about the cache, is to simply specify cache_discovery=False when performing discovery.build() -- this avoids the broken code path.

However, I think we shouldn't disable the cache discovery, just in case the library's developers fix this in the future. What do you think?